src/documentation/documentation.scss

```diff
@@ -1,2 +1,38 @@
+@import "styles/styles.scss";
 .documentsContainer {
```

Comment:
The layout of the page does not conform to the doc.  (For example: Is missing the header)

Reply:
still don't have it

![image](https://user-images.githubusercontent.com/1821666/120407222-edff8500-c309-11eb-8b80-5166b5379cd1.png)

Reply:
This is what the branch currently has @dkent600 

![screenshot](https://user-images.githubusercontent.com/19796652/120457360-4a0bed00-c38e-11eb-8936-005d0d3a2919.png)

Reply:
This is what the branch currently has 
<img width="1232" alt="Screenshot 2021-06-02 at 10 37 07" src="https://user-images.githubusercontent.com/19796652/120457600-7c1d4f00-c38e-11eb-9c7b-2c34ce410b6b.png">

Reply:
The layout now matches. Please re-review

Reply:
Doesn't match.  There should not be a different background color and there is too much column whitespace between "Documentation" and "Title 1"

![image](https://user-images.githubusercontent.com/1821666/120471836-71e55b80-c362-11eb-881f-c06df3f449a1.png)

Reply:
Okay so here is the adjusted documentation page, should match the mocks now.
<img width="1413" alt="Screenshot 2021-06-02 at 12 51 06" src="https://user-images.githubusercontent.com/19796652/120475380-33bb5c80-c3a1-11eb-8e93-443dd48dd145.png">